data/projects/k/kiwi-news-attestate.yaml

```diff
@@ -6,26 +6,28 @@ github:
   - url: https://github.com/attestate/awesome-kiwinews
   - url: https://github.com/attestate/delegator2
 blockchain:
-  - address: "0xee324c588ceF1BF1c1360883E4318834af66366d"
```

Comment:
This is the address listed as the payout in the [RPGF3 application](https://vote.optimism.io/retropgf/3/application/0x50565b1e518ab91b14ef50fdab91a63623fb5951e96c18918e36be5aeb6fd297), no?

Reply:
yeah but it is my personal address as we don't have a optimism gnosis safe at the moment. But when it comes to tracking activity on OP my address doesn't really matter. What matters are the contracts that users interact with

Reply:
OK anyways I added it back on